Lib/symtable.py

```diff
@@ -119,8 +119,8 @@ class Function(SymbolTable):
     __globals = None
 
     def __idents_matching(self, test_func):
-        return tuple(ident for ident in self.get_identifiers()
-                     if test_func(self._table.symbols[ident]))
+        return tuple([ident for ident in self.get_identifiers()
+                      if test_func(self._table.symbols[ident])])
 
```

Comment:
The list comprehension is not necessary and doesn't improve the code. This should stay as-is.